ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/jline/JlineSimpleCompleter.java

```diff
@@ -85,4 +88,5 @@ public void complete(LineReader reader, ParsedLine line, List<Candidate> candida
     private boolean hasPropertyPrefix(ParsedLine line, String propertyName) {
         return line.line().trim().startsWith(PropertiesLoader.getProperty(propertyName));
     }
+
```

Comment:
Unnecessary line

Reply:
Removed in a1286fe5ba1e38ae8718140c18a16fadd90542e6